.nix/config.nix

```diff
@@ -79,11 +79,14 @@
     ## Run on push on following branches (default [ "master" ])
     # push-branches = [ "master" "branch2" ];
 
-    "master".coqPackages = {
-      coq.override.version = "master";
-      coq-elpi.override.version = "coq-master";
-      hierarchy-builder.override.version = "master";
-      mathcomp.override.version = "master";
+    "master" = {
+      coqPackages = {
+        coq.override.version = "master";
+        coq-elpi.override.version = "coq-master";
+        hierarchy-builder.override.version = "master";
+        mathcomp.override.version = "master";
+      };
+      ocamlPackages.elpi.override.version = "v1.18.1";
```

Comment:
this is a workaround, but we need to upgrade elpi in mainstream nixpkgs

Reply:
C.f., https://github.com/coq-community/coq-nix-toolbox/pull/191